src/components/derivation/Create/index.tsx

```diff
@@ -13,22 +16,49 @@ function DerivationCreate() {
     // it's closed, so you don't reopen it and have your previous
     // selections still selected, which would be unexpected.
     const [newCollectionKey, setNewCollectionKey] = useState(0);
+    const [showConfirmation, setShowConfirmation] = useState(false);
+
+    const closeDialog = () => {
+        navigate(authenticatedRoutes.collections.fullPath);
+        setShowConfirmation(false);
+        setNewCollectionKey((k) => k + 1);
+    };
 
     return (
         <Dialog
             open
-            fullWidth
+            fullWidth={!showConfirmation}
             maxWidth="lg"
-            onClose={() => {
-                navigate(authenticatedRoutes.collections.fullPath);
-                setNewCollectionKey((k) => k + 1);
-            }}
+            onClose={closeDialog}
+            aria-labelledby={ARIA_LABEL_ID}
         >
-            <DialogTitle>
+            <DialogTitleWithClose id={ARIA_LABEL_ID} onClose={closeDialog}>
                 <FormattedMessage id="newTransform.modal.title" />
-            </DialogTitle>
+            </DialogTitleWithClose>
             <DialogContent>
-                <TransformationCreate key={newCollectionKey} />
+                <Collapse in={showConfirmation}>
+                    <AlertBox
+                        short
+                        severity="info"
+                        title={
+                            <Typography>
+                                <FormattedMessage id="newTransform.info.gitPodWindowTitle" />
+                            </Typography>
+                        }
+                    >
+                        <FormattedMessage id="newTransform.info.gitPodWindowMessage" />
+                    </AlertBox>
+                </Collapse>
```

Comment:
Show some kind of confirmation when the window has been opened in case a user does not directly go to GitPod.